addons/project/views/project_views.xml

```diff
@@ -614,11 +604,15 @@
                                             </div>
                                         </div>
                                     </div>
-                                    <a class="o_kanban_manage_toggle_button o_left" href="#" groups="base.group_user"><i class="fa fa-ellipsis-v" role="img" aria-label="Manage" title="Manage"/></a>
-                                    <span class="o_right"><field name="is_favorite" widget="boolean_favorite" nolabel="1" force_save="1" /></span>
+                                    <a class="o_kanban_manage_toggle_button o_display_on_kanban_card_hover" href="#" groups="base.group_user">
+                                        <i class="fa fa-ellipsis-v" role="img" aria-label="Manage" title="Manage"/>
+                                    </a>
+                                    <span t-att-class="record.is_favorite.raw_value ? '':'o_display_on_kanban_card_hover'">
+                                       <field name="is_favorite" widget="boolean_favorite" nolabel="1" force_save="1" />
+                                    </span>
                                 </div>
                                 <div class="o_kanban_record_bottom">
-                                    <div class="oe_kanban_bottom_left">
+                                    <div class="oe_kanban_bottom_left ml-2">
```

Comment:
I would rather add this rule in project_dashboard.scss [here](https://github.com/odoo/odoo/blob/fd6430eefbcc66d6e30304d3a8fd8b6a9f3b71b1/addons/project/static/src/scss/project_dashboard.scss#L17) than adding the ml-2 class:
```css
            &:first-child {
                margin: 0 5px 0 0;
                padding-left: 12px
            }
```
What do you think?

Reply:
Sure, no problem, it works as well. Is it usually recommended to avoid applying css classes inline?

Reply:
no not at all and I would encourage adding class instead of custom css. But here, as there was already css applied on the items, I would rather do it there + is is now applied at a lower level and not on the whole oe_kanban_bottom_left. If you look in the main part of the kanban it is the same it in not the whole block but a lower part. So I find it better to do the same here, otherwise the two blocks would no more be aligned.